stacks-coordinator/src/bitcoin_node.rs

```diff
@@ -165,11 +168,14 @@ impl LocalhostBitcoinNode {
         let json_rpc =
             ureq::json!({"jsonrpc": "2.0", "id": "stx", "method": method, "params": params});
 
-        let url = path
-            .map(|path| Cow::Owned(format!("{}{}", &self.bitcoind_api, path)))
-            .unwrap_or(Cow::Borrowed(&self.bitcoind_api));
+        let url = if let Some(path) = path {
+            Cow::Owned(self.bitcoind_api.join(path)?)
+        } else {
+            Cow::Borrowed(&self.bitcoind_api)
+        };
 
-        let response = ureq::post(&url)
+        println!("Calling {} ...", url);
```

Comment:
Nit: Remove or change to debug logging
```suggestion
        tracing::debug!("Calling url: {}.", url);
```

Reply:
Good catch :)